src/Mailer/Transport/SmtpTransport.php

```diff
@@ -169,7 +169,7 @@ public function getLastResponse(): array
      * Send mail
      *
      * @param \Cake\Mailer\Message $message Message instance
-     * @return array
+     * @return array<string, string>
```

Comment:
Since phpstorm now understands array shapes this can be changed to the more accurate `array{headers: string, message: string}` and the `@psalm-return` below removed. That would avoid having to ignore errors reported by phpstan/psalm.

Reply:
Also on option, sure.
Would be the first time we use that inside normal tags then however.

Reply:
Using `array<>` form was once a first too :)

Reply:
I feel like this could be a follow up, as there are other issues that might need solving then, e.g.

> Property Cake\Mailer\Transport\SmtpTransport::$_content (array{headers: string, message: string}) does not accept default value of type array{}.

Reply:
I adjusted now the return type, that seems fine. I cannot adjust the property, however, due to the error above.